src/client-side-encryption/auto_encrypter.ts

```diff
@@ -379,7 +387,8 @@ export class AutoEncrypter {
       promoteValues: false,
```

Comment:
(hidden in the diff) - should we also apply this option to the MongoClient instantiated in the constructor of the AutoEncrypter?